### PR TITLE
add Nuget.Commands and Nuget.Credentials 6.2.2

### PR DESCRIPTION
**Pull request by oleksandr-didyk**
Fixes product pre-builts introduced by https://github.com/dotnet/installer/pull/16592 by adding `Nuget.Commands` and `Nuget.Credentials` `6.2.2`

### CODE REVIEW

File: log.txt
Comment (mthalman) [inline log.txt:1]: Delete this file

**Comment by MichaelSimons** [comment]
Have you verified Arcade is not loading these?  Given arcade is a toolset repo, there is a good chance they are.

File: log.txt
Comment (oleksandr-didyk) [reply log.txt:1]: My bad, removed

**Comment by oleksandr-didyk** [comment]
> Have you verified Arcade is not loading these? Given arcade is a toolset repo, there is a good chance they are.

Viktor disabled bundling of `Nuget.*` packages with Arcade's projects as mentioned in https://github.com/dotnet/source-build-reference-packages/pull/673#issuecomment-1569508658

**Comment by MichaelSimons** [comment]
> Viktor disabled bundling of Nuget.* packages with Arcade's projects as mentioned in https://github.com/dotnet/source-build-reference-packages/pull/673#issuecomment-1569508658. That was the original reason we could use SBRPed Nuget for Arcade.

Yes but technically it is possible for a nuget package to reference 6.2.2 which would load the SBRP.

**Comment by oleksandr-didyk** [comment]
> I will test it out with the VMR just to be safe

The build passed, Nuget* pre-builts are not present

**Review by MichaelSimons — APPROVED** [review]

**Pull request merged**